docs/user-guide/logging/sdl.md

```diff
@@ -1,14 +1,17 @@
 ---
 assignees:
-- lavalamp
-- satnam6502
+- crassirostris
+- piosz
 
 ---
 
-A Kubernetes cluster will typically be humming along running many system and application pods. How does the system administrator collect, manage and query the logs of the system pods? How does a user query the logs of their application which is composed of many pods which may be restarted or automatically generated by the Kubernetes system? These questions are addressed by the Kubernetes **cluster level logging** services.
+Before reading this page, it's recommended to take a look at [a general description](/docs/user-guide/logging/overview) of how logging works in kubernetes.
 
-Cluster level logging for Kubernetes allows us to collect logs which persist beyond the lifetime of the pod's container images or the lifetime of the pod or even cluster. In this article we assume that a Kubernetes cluster has been created with cluster level logging support for sending logs to Google Cloud Logging. After a cluster has been created you will have a collection of system pods running in the `kube-system` namespace that support monitoring,
-logging and DNS resolution for names of Kubernetes services:
+In this article we assume that a Kubernetes cluster has been created with cluster-level logging support for sending logs to Stackdriver Logging.
```

Comment:
"This article assumes that you have created a Kubernetes cluster with cluster-level logging support for sending logs to Stackdriver Logging."